index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Baroka'a Portofolio</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <img class="logo" src="images/image_geometry.png" alt="logo">
+        <img class="toolbar" src="images/ic_menu.png" alt="">
+      </nav>
+      <div class="headline">
+        <h1 class="hero">Irotori Baroka</h1>
+        <p>
+            Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.
+        </p>
+        <button type="button" class="header-button">Start collaboration</button>
+      </div>
```

Comment:
- [ ] Kindly move this section outside of the header tag and wrap it with a `section` tag.
- Please make sure to stick to the Figma design as much as possible, which includes: 
   - [ ] Adjusting the spacing between the header title and header text.
   - [ ] Adjusting the font of the header text to match the design.
   - [ ] Matching the design and font of the button to match the Figma design.

| Figma design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/62290545/158577247-c693dc75-c5b4-40cc-af01-e236a8964965.png) | ![image](https://user-images.githubusercontent.com/62290545/158577377-8dec9e4f-2ec2-4c1f-9b09-08bdddc53729.png) |